test/helpers/helper_functions.js

```diff
@@ -23,6 +25,47 @@ function readRequestQueue() {
     var a = JSON.parse(fs.readFileSync(reqDir, "utf-8")).cly_queue;
     return a;
 }
+function readBulkEventQueue() {
```

Comment:
seems like you can use the methods above. You can set the default dir and if anything is given you can use that.